reference/api/settings.md

```diff
@@ -1,117 +1,116 @@
-# All settings
+---
+sidebarDepth: 2
+---
 
-The `/settings` route allows you to customize search settings for the given index. It is possible to modify all of an index's settings at once using the [`update settings` endpoint](#update-settings), or modify each one individually using the child routes.
+# Settings
 
-These are the reference pages for the child routes:
+The `/settings` route allows you to customize search settings for the given index. You can either modify all of an index's settings at once using the [update settings endpoint](#update-settings), or modify each one individually using the child routes.
 
-- [Displayed attributes](/reference/api/displayed_attributes.md)
-- [Distinct attribute](/reference/api/distinct_attribute.md)
-- [Faceting](/reference/api/faceting.md)
-- [Filterable attributes](/reference/api/filterable_attributes.md)
-- [Pagination](/reference/api/pagination.md)
-- [Ranking rules](/reference/api/ranking_rules.md)
-- [Searchable attributes](/reference/api/searchable_attributes.md)
-- [Sortable attributes](/reference/api/sortable_attributes.md)
-- [Stop-words](/reference/api/stop_words.md)
-- [Synonyms](/reference/api/synonyms.md)
-- [Typo tolerance](/reference/api/typo_tolerance.md)
+For a conceptual overview of index settings, refer to our [indexes guide](/learn/core_concepts/indexes.md#index-settings).
 
-To learn more about settings, refer to our [dedicated guide.](/learn/configuration/settings.md)
+## Settings object
 
-::: warning
-When you update a setting, you overwrite its default value. Use the `DELETE` route to reset any setting to its original value.
-:::
+By default, the settings object looks like this. All fields are modifiable.
 
-## Get settings
+```json
+{
+  "displayedAttributes":[
+    "*"
+  ],
+  "searchableAttributes":[
+    "*"
+  ],
+  "filterableAttributes":[],
+  "sortableAttributes":[],
+  "rankingRules":
+  [
+    "words",
+    "typo",
+    "proximity",
+    "attribute",
+    "sort",
+    "exactness"
+  ],
+  "stopWords":[],
+  "synonyms":{},
+  "distinctAttribute":null,
+  "typoTolerance":{
+    "enabled":true,
+    "minWordSizeForTypos":{
+      "oneTypo":5,
+      "twoTypos":9
+    },
+    "disableOnWords":[],
+    "disableOnAttributes":[]
+    },
+  "faceting":{
+      "maxValuesPerFacet":100
+      },
+  "pagination":{
+      "maxTotalHits":1000
+      }
+}
+```
 
-<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings" />
+To learn more about any one of these fields, click on its name in the sidebar.
```

Comment:
I find it a bit clunky when a text directly refers to a UI element that's outside the regular content flow. It might also be a bit not future-proof if we change the content on the sidebar menu somewhere else on the screen on the new design. 🤔